dolphinscheduler-ui-next/src/views/projects/task/constants/task-type.ts

```diff
@@ -93,5 +94,8 @@ export const TASK_TYPES_MAP = {
   EMR: {
     alias: 'AmazonEMR',
     helperLinkDisable: true
+  },
+  ZEPPELIN: {
+    alias: 'ZEPPELIN'
```

Comment:
Is there a instruction for the new task type. If not, should set the helperLinkDisable to true.

Reply:
No instructions yet, will set the helperLinkDisable. Thx